src/tracks/playabletrack/wavetrack/ui/PitchAndSpeedDialog.cpp

```diff
@@ -269,13 +269,8 @@ PitchAndSpeedDialog& PitchAndSpeedDialog::SetFocus(
          nullptr;
    if (item)
       item->SetFocus();
-#ifndef _WIN32
-   // On macOS, just calling `SetFocus` isn't enough, but on Windows, changing
-   // visibility to false and then back to true is ugly, so don't do this there.
-   // TODO: Linux ?
-   Show(false);
-#endif
-   Show(true);
+   wxDialog::Show(true);
```

Comment:
But why do we still need both Hide/Show cycle and Raise? I expected that Raise would be sufficient alone

Reply:
Hide/Show cycle? We are only calling `Show(true)`. If that's what you meant: maybe, but does it hurt?

Reply:
Hmm, I misread the diff, sorry